terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,74 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                 = "order-service" # this must match the stack name (cluster) the service deploys into
+  name_prefix                = "${local.stack_name}-${var.environment}"
+  global_prefix              = "global-${var.environment}"
+  service_name               = "document-signing-api"
+  container_port             = "8080"
+  eric_port                  = "10000"
+  docker_repo                = "document-signing-api"
+  kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority  = 181
+  lb_listener_paths          = ["/document-signing-api/*", "/document-signing/*"]
```

Comment:
stepping out of my comfort zone here - but won't this make the healthcheck available through the lb?

Reply:
looks like the api's only functional endpoint apart from the healthcheck is at: /document-signing/sign-pdf

Reply:
If that's definitely the only path needed then I can update! Is there a reason for /document-signing-api/* being in the routes.yaml?

Reply:
I guess because we used to want to be able to access the healthcheck via a http request?